examples/static-hbase-logs/s3.tf

```diff
@@ -22,7 +22,7 @@ resource "aws_s3_bucket_object" "sample_bootstrap_script" {
   depends_on = [local_file.cloudwatch-install]
 
   bucket                 = module.emr-rootdir-bucket.bucket_name
-  key                    = "bootstrap-actions/cloudwatch-install.sh"
-  source                 = "${path.module}/files/cloudwatch-install.sh"
+  key                    = "bootstrap-actions/cloudwatch-install-hbase.sh"
+  source                 = "${path.module}/tempfiles/cloudwatch-install-hbase.sh"
```

Comment:
we should make it agnostic of the applications we use, cloudwatch-install-emr sounds better

Reply:
I will do it in the next iteration.